Makefile

```diff
@@ -2,6 +2,7 @@ TARGET_DIR=bin
 GOBUILD=go build
 MAIN_DIR=cmd
 BINARY_NAME=go_ta_pago_bot
+MIGRATIONS_DIR=database/migrations
```

Comment:
please rename the migration directory to: **internal/db/migrations**. This one will be soon uploaded